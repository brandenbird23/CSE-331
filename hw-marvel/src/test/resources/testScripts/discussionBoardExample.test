# Tries to find path that should be 3 edges via discussion post

LoadGraph marvel marvel.csv

FindPath marvel SILVER-SURFER/NORRIN G'RATH