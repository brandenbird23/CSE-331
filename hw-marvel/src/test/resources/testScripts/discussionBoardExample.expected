# Tries to find path that should be 3 edges via discussion post

loaded graph marvel

path from SILVER-SURFER/NORRIN to G'RATH:
SILVER-SURFER/NORRIN to DAGGER/TANDY-BOWEN via IG-6
DAGGER/TANDY-BOWEN to KILLRAVEN/JONATHAN-R via AA2-18
KILLRAVEN/JONATHAN-R to G'RATH via AA2-35